src/detect-byte.c

```diff
@@ -25,6 +25,10 @@
 #include "detect-byte.h"
 #include "detect-byte-extract.h"
 #include "detect-bytemath.h"
+#include "detect-engine.h"
```

Comment:
Why do we need to add includes here if we do not change code ?

Reply:
This was added because `rust-bindings.h` was needed ... however, i'll remove it and include `rust.h` instead of `rust-bindings.h`

Reply:
How come rust-bindings.h was needed if nothing was changed in this C file ?